server/store/user_store.go

```diff
@@ -268,6 +282,14 @@ func (s *pluginStore) StoreUserActiveEvents(mattermostUserID string, events []st
 	return kvstore.StoreJSON(s.userKV, mattermostUserID, u)
 }
 
+func (index UserIndex) ToDTO() (result []UserShortDTO) {
+	for _, u := range index {
+		result = append(result, u.ToDTO())
+	}
+
+	return
```

Comment:
Unnecessary return

Reply:
Why?

Reply:
Oh I see the return value var is in the signature. I suppose I'm not used to seeing that too often